onnxruntime/test/shared_lib/custom_op_utils.h

```diff
@@ -14,30 +14,21 @@ struct Input {
   std::vector<float> values;
 };
 
-struct OrtTensorDimensions : std::vector<int64_t> {
-  OrtTensorDimensions(Ort::CustomOpApi ort, const OrtValue* value) {
-    OrtTensorTypeAndShapeInfo* info = ort.GetTensorTypeAndShape(value);
-    std::vector<int64_t>::operator=(ort.GetTensorShape(info));
-    ort.ReleaseTensorTypeAndShapeInfo(info);
-  }
-};
-
 struct MyCustomKernel {
-  MyCustomKernel(Ort::CustomOpApi ort, const OrtKernelInfo* /*info*/, void* compute_stream)
-      : ort_(ort), compute_stream_(compute_stream) {
+  MyCustomKernel(const OrtKernelInfo* /*info*/, void* compute_stream)
```

Comment:
nit: Can remove `OrtKernelInfo` from kernel constructors if not used. Would just have to update the associated `CreateKernel` functions to not pass in the `info` arg. I saw this a few times in this file.

Reply:
Talked offline. Will leave OrtKernelInfo args as an example.